src/notebooks/student_logins.ipynb

```diff
@@ -0,0 +1,277 @@
+{
+ "cells": [
```

Comment:
Can you add a markdown cell at the top describing the purpose of this notebook, and the data that it will analyze?